eZ/Bundle/PlatformInstallerBundle/src/Installer/DbBasedInstaller.php

```diff
@@ -67,4 +73,34 @@ protected function runQueriesFromFile($file)
             $this->db->exec($query);
         }
     }
+
+    /**
+     * Get DBMS-specific SQL data file path.
+     *
+     * @param string $relativeFilePath SQL file path relative to {baseDir}/{dbms} directory
+     *
+     * @return string absolute existing file path
+     *
+     * @throws \eZ\Publish\Core\Base\Exceptions\InvalidArgumentException
+     */
+    protected function getDatabasePlatformSpecificSQLFile($relativeFilePath)
```

Comment:
maybe even set as final

Reply:
And maybe we should make this method available on 6.x as well, so installers can work across both versions if package maintainer wants.

Reply:
That would be a good idea, yes.